test/regression.js

```diff
@@ -18,11 +19,21 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const width = 960;
 const height = 720;
 
+const FILE_PATTERN = new RegExp('.*');
+let configFileName;
+const DEFAULT_CONFIG = {
+  floatPrecision: 4,
+};
+const CONFIG = configFileName ? readConfigFile(configFileName) : DEFAULT_CONFIG;
```

Comment:
I'd much prefer if we didn't use config files or create output files ourselves.

For input, we can read from CLI as you allude to in your PR description, or an environment variable.

For output, we should keep it to STDOUT, and the contributor can choose to pipe or redirect it to whatever tool/file they want if they want to process it or present it differently.